draft-ietf-httpbis-client-hints.md

```diff
@@ -247,13 +247,13 @@ If Downlink occurs in a message more than once, the minimum value should be used
 The "Save-Data" request header field is a token that indicates client's preference for reduced data usage, due to high transfer costs, slow connection speeds, or other reasons.
 
 ~~~ abnf7230
-  Save-Data = "Save-Data" ":" sd-token *( ";" [ sd-parameter ] )
+  Save-Data = "Save-Data" ":" sd-token *( ";" [ sd-extension ] )
 
   sd-token = "on"
-  sd-parameter = token
+  sd-extension = token
 ~~~
 
```

Comment:
The header field name (and the colon) should not be part of the ABNF.

Also, the ABNF does not allow any whitespace between "on" and potential extensions, is that intentional? (just checking...)